contrib/syslog-debun

```diff
@@ -456,6 +546,172 @@ debun_extra_debian () {
 	md5sum --quiet -c /var/lib/dpkg/info/syslog-ng*.md5sums && echo Package files are intact
 	echo "list syslog-related packages"
 	dpkg -l |grep syslog > ${tmpdir}/deb.packages
+}
+
+debun_extra_redhat () {
+	printf "\nRedhat specific checks\n"
+	echo "Check package files integrity"
+	rpm -V syslog-ng-premium-edition && echo "syslog-ng-premium-edition: Package files are intact"
+	rpm -V syslog-ng-premium-edition-client && echo "syslog-ng-premium-edition-client: Package files are intact"
+	rpm -V syslog-ng-premium-edition-compact && echo "syslog-ng-premium-edition-compact: Package files are intact"
+	echo "list syslog-related packages"
+	rpm -qa |grep syslog > ${tmpdir}/rpm.packages
+}
+
+debun_extra_slackware () {
+	printf "\nSlackware Linux specific checks\n"
+	echo "list syslog-related packages"
+	find /var/log/packages -name "*sys*log*" | while read -r FILE; do echo "${FILE##*/}"; done > ${tmpdir}/pkg.packages
+}
+
+debun_extra_suse() {
+	printf "\nSuSE specific checks\n"
+	echo "Check package files integrity"
+	rpm -V syslog-ng-premium-edition && echo "syslog-ng-premium-edition: Package files are intact"
+	rpm -V syslog-ng-premium-edition-client && echo "syslog-ng-premium-edition-client: Package files are intact"
+	rpm -V syslog-ng-premium-edition-compact && echo "syslog-ng-premium-edition-compact: Package files are intact"
```

Comment:
Same here, what I wrote above for rpm.
It seems pretty much the same code.
Do not copy them! Consider to put the rpm integrity checking into a separate function, and call them from both place.
Btw.: You may consider to write package validity checking part for .deb systems aswell.